### ISSUE TITLE
cannot install

### ISSUE THREAD

**Issue by philippmossier** [CosmicNvim/CosmicNvim#80]
Hi, thanks for this awesome neovim environment I really like it. I used it a year before which worked fine with 0.7 but now as I tried the newest version of CosmicNvim with nvim 0.8 I am not able to install it.

# Steps to reproduce:

## Machine preferences
Using Ubuntu inside WSL2
![image](https://user-images.githubusercontent.com/49983520/176427999-cbaade78-e5a5-4da5-8bd0-acf6ced84e8b.png)
node v16.13.1
zsh v5.8

## Steps before cloning CosmicNvim

1. Delete nvim cache and share folder:
```
rm -rf .local/share/nvim
rm -rf .cache/nvim
```

2. Remove current neovim and Install latest nightly neovim
```
mkdir -p ~/.local/bin && \rm -f ~/.local/bin/nvim && curl -L https://github.com/neovim/neovim/releases/download/nightly/nvim.appimage -o ~/.local/bin/nvim && chmod u+x ~/.local/bin/nvim
```

3. check version
```
nvim -v
NVIM v0.8.0-dev+513-g01fc5097d
```

4. Clone cosmic vim
```
cd .config
git clone git@github.com:CosmicNvim/CosmicNvim.git nvim
```

5. open nvim and install plugins
```
nvim +CosmicReloadSync
```

# I get this errors
![image](https://user-images.githubusercontent.com/49983520/176429877-44eb9973-1d1a-4b4b-b520-156c2caa548c.png)

### After pressing enter it looks like 3 plugins failed to install (which normally installs them succesfully after a second try with :PackerSync)
![image](https://user-images.githubusercontent.com/49983520/176430041-e4154cfc-8e87-4f6a-82a3-b67c8122279f.png)

### after pressing `enter` 2 times, I see this:
![image](https://user-images.githubusercontent.com/49983520/176426651-4a3dd50c-5521-48bb-86a9-233a43f6f947.png)

### after pressing `esc` I see this (it looks like installed now => [nvim-treesitter] [12/12])
![image](https://user-images.githubusercontent.com/49983520/176426930-61508355-f71d-4ce7-999e-fb06e71b059d.png)

### after quitting neovim and running neovim again with `nvim .` 

![image](https://user-images.githubusercontent.com/49983520/176427085-4eb4c791-49e5-4bf2-8389-8bc41fac9e41.png)

I tried this a few times, with slighty different approaches with no success.

**Comment by philippmossier** [comment]
After running packerSync I see the 3 packages which failed, are now sucessfully installed. So all required packages should be installed.

windwp/nvim-autotag
nvim-treesitter/nvim-treesitter/
nvim-treesitter/nvim-treesitter-refactor

**Comment by philippmossier** [comment]
fixed after new nighlty version: NVIM v0.8.0-dev+519-g6f6286e4f
